### PR TITLE
Added implementation for EventConfirmation.tsx

### PR DESCRIPTION
**Pull request by snehar27**
## Summary

Implemented EventConfirmation.tsx.

Screenshots:

image
image

image
image

## Testing

A Test.tsx file may created with the following implementation:

```import React from "react";
import EventConfirmation from "../components/organisms/EventConfirmation";

const Test = () => {
  return (
    <EventConfirmation
      title="test1"
      eventid="0000"
      supervisor="test1"
      location="test"
      capacity={2}
      datetime="today"
      action="rsvp"
    />
  );
};

export default Test;
```

Then run `yarn run start` and navigate to `http://localhost:3000/Test#`

### CODE REVIEW

**Comment by akinfelami** [comment]
This doesn't meet the standards for making a PR. The description is vague. How do I test it? Are there any notes?

File: frontend/src/components/molecules/CardList.tsx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import React, { ReactElement, Children } from "react";
-import EventCard from "@/components/molecules/EventCard";
+// import EventCard from "@/components/molecules/EventCard";
```
Comment (akinfelami) [inline frontend/src/components/molecules/CardList.tsx:2]: It looks like the only change to this file is this commented out line. Remove it if it is not of any use

**Review by akinfelami — CHANGES_REQUESTED** [review]
Hi! I don't know how to see the changes you made. Can you make the PR more descriptive and offer a way to test it? Then I can offer some more feedback on the code.

**Review by Y1chenYao — CHANGES_REQUESTED** [review]
- The icon/headline/link should be styled (centered/grey color, etc). 
image

- Some components can be replaced with `EventDetails.tsx` as a placeholder. 
- Rather than `if "rsvp" else if "not rsvp"`, I recommend using a boolean value to represent `rsvp` action. During testing, I got the error of possibly null return value, which is due to the fact that if we have an action that is neither "rsvp" nor "not rsvp", nothing will be returned.

File: frontend/src/components/organisms/EventConfirmation.tsx [hunk 2]
```diff
@@ -10,8 +22,77 @@ interface EventConfirmationProps {
...
+            </Grid>
+            <Grid item xs={3}>
+              <Button>{title}</Button>
+            </Grid>
+          </Grid>
+        </Grid>
+        <p>
+          If you can no longer attend,{" "}
+          <a href="#" rel="noreferrer">
+            cancel here
```
Comment (bananahz) [inline frontend/src/components/organisms/EventConfirmation.tsx:60]: add a space here

Comment (bananahz) [inline frontend/src/components/organisms/EventConfirmation.tsx:60]: add a space after cancel here

**Comment by bananahz** [comment]
Building off Yichen's comment, on figma if you go to dev mode, theres exact specifics for styling (rem, color, etc.) :)

**Review by bananahz — CHANGES_REQUESTED** [review]

File: frontend/src/pages/Test.tsx
Comment (bananahz) [inline frontend/src/pages/Test.tsx:1]: since this page is for testing purposes, we can remove it from the PR too since we dont want Test to be pushed

**Review by Y1chenYao — APPROVED** [review]
Looks pretty good!

**Review by bananahz — APPROVED** [review]
lgtm :)

**Review by akinfelami — APPROVED** [review]
lgtm

**Review by jasozh — APPROVED** [review]
Good work. Things to note:
- Remember to delete unused imports
- Although the Figma shows a gray box, remember that at the end of the day, we're passing in the `EventDetails` component into the slot where the gray box is. Use the gray box as a reference while testing (and leave it in for the PR), but also include the actual component (in this case `EventDetails`) in the future.

**Pull request merged**